### DISCUSSION TITLE
Seeking clarification about the new hidden channels thumbnail retrieval feature

### DISCUSSION THREAD

**Discussion by Flopblopper** [FreeTubeApp/FreeTube#5028] [Q&A]
I understand that the thumbnail URLs for hidden channels are stored in the settings.db file. I was wondering if someone could clarify whether the actual thumbnails are cached, or if they are retrieved each time you open the settings panel. I have a large amount of hidden channels, and I want to make sure that I'm not frequently fetching thumbnails for them, since that'd probably get me rate limited pretty quickly.

Thank you

**Comment by kommunarr** [comment]
Yes, these are cached until you reload or restart FreeTube. In case you're curious in the future, you can open up the Chrome DevTools (In Windows, <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>) and navigate to the Network tab to track which requests are being made as you navigate throughout the app.
